node/miden-node.toml

```diff
@@ -4,6 +4,7 @@
 # port defined as: sum(ord(c)**p for (p, c) in enumerate('miden-block-producer', 1)) % 2**16
 endpoint = { host = "localhost", port = 48046 }
 store_url = "http://localhost:28943"
+verify_tx_proofs = true
```

Comment:
Let's add a comment here to explain what `verify_tx_proofs` does. This should be a user-facing comment.